Core/include/Acts/Propagator/detail/LoopProtection.hpp

```diff
@@ -18,11 +18,17 @@ namespace detail {
 template <typename path_aborter_t, typename propagator_state_t,
           typename stepper_t>
 void setupLoopProtection(propagator_state_t& state, const stepper_t& stepper,
-                         path_aborter_t& pathAborter, const Logger& logger) {
+                         path_aborter_t& pathAborter, bool releaseLimit,
+                         const Logger& logger) {
   if (!state.options.loopProtection) {
     return;
   }
 
+  if (releaseLimit) {
+    pathAborter.internalLimit =
+        state.options.direction * std::numeric_limits<double>::max();
```

Comment:
You recover the sign here for the internal limit in this case?

Reply:
yeah not happy about the sign of this property but currently we use a sign for the accumulated path length and this property is compared to it and it seems natural to keep the sign on this one too